ui/mirage/models/auth-method.ts

```diff
@@ -0,0 +1,13 @@
+import { Model } from 'miragejs';
+import { OIDCAuthMethod } from 'waypoint-pb';
+
+export default Model.extend({
+  toProtobuf(): OIDCAuthMethod {
+    let result = new OIDCAuthMethod();
+
+    result.setDisplayName(this.displayName);
+    result.setKind(0);
```

Comment:
Seems like this data is managed by the factory?

```suggestion
    result.setKind(this.kind);
```